stdlib/ballerina-database/src/main/java/org/ballerinalang/database/sql/SQLDataIterator.java

```diff
@@ -463,48 +465,23 @@ private void handleDateValue(int index, BStruct bStruct, RegistryIndex stringReg
         }
     }
 
-    private void handleBlobValue(BStruct bStruct, RegistryIndex refRegIndex, RegistryIndex blobRegIndex, int index)
-            throws SQLException {
+    private void handleBinaryValue(BStruct bStruct, RegistryIndex refRegIndex, RegistryIndex blobRegIndex, int index,
+     byte[] bytes) {
         BStructType.StructField[] structFields = getStructFields();
         int fieldTypeTag = getFieldTypeTag(structFields, index);
```

Comment:
Shall we remove both below lines from all the methods and move them to the switch case level?
```
BStructType.StructField[] structFields = getStructFields();
int fieldTypeTag = getFieldTypeTag(structFields, index);
```